### ISSUE TITLE
Ledger LP withdrawal "transaction fee needs to be covered by your balance"

### ISSUE THREAD

**Issue by jeffdgr8** [asgardex/asgardex-desktop#448]
## Prerequisites
Ledger LP position (not sure if specific to Ledger addresses, but haven't tested with non-Ledger addresses)

## Expected Behavior
Able to withdraw, assuming large enough amount.

## Current Behavior
No matter what percentage chosen to withdraw, the error "TRANSACTION FEE ᚱ 0.08 NEEDS TO BE COVERED BY YOUR BALANCE (CURRENTLY ᚱ 0)." is shown and the WITHDRAW button is disabled.

## Steps to Reproduce
1. Go to manage LP
2. Select Ledger for both RUNE and asset
3. Select withdraw tab
4. Select large enough percentage that RUNE balance should cover transaction fee

## Possible Solution
Unknown

## Your Environment - Release Version (skip if not using a release version)
- ASGARDEX desktop version: V1.22.7
- Operating system and version: Windows 11

## Your Environment - Development Version (skip if not using a development version)

- Node version:

- Yarn version:

- Branch:

- Operating system and version:

**Comment by Thorian1te** [comment]
asgardex doesn't support asymmetric LP positions so if you have one of these a withdrawal won't work. 
If you want to add or remove a asymtrical position you have to use another provider such as thorswap. 

You are able to send a transaction to thorchain if you know how to handle inbound addresses? 
using `WITHDRAW:POOL:10000 — dual-sided 100% withdraw liquidity. If a single-address position, this withdraws single-sidedly instead`

**Comment by jeffdgr8** [comment]
This is with dual-sided LP positions.

**Comment by Thorian1te** [comment]
and you have a rune balance?

**Comment by jeffdgr8** [comment]
Yes, there is also RUNE in the same Ledger wallet address as the LP position.

**Comment by Thorian1te** [comment]
hey looking into this issue,im not getting your error, confirm that it looks like this with both asset having the ledger tag ? 
<img width="557" alt="image" src="https://github.com/user-attachments/assets/d24f841b-19ee-4c47-9c75-7d184abe9039">

**Comment by jeffdgr8** [comment]
It does this for all assets I've tested. V1.22.8
![Screenshot 2024-11-17 185150](https://github.com/user-attachments/assets/d0a9d71e-8afd-4335-ab96-c8d9de9abb7a)

**Comment by Thorian1te** [comment]
And you can confirm that the ledger Rune address holds more than say 1 rune.

**Comment by Thorian1te** [comment]
if you have a balance for rune and its not recognising it I am not sure why this is happening I can't see it in dev state. 
I have put in this extra content example below. 
<img width="654" alt="image" src="https://github.com/user-attachments/assets/7e757573-e381-4231-8d63-f5e79baf4e0c">
If you need to do a withdraw use the memo and go to RUNE custom deposit for the address you used in the LP position. 
all it tells thornode is, withdraw this % of the LP.

**Comment by jeffdgr8** [comment]
> And you can confirm that the ledger Rune address holds more than say 1 rune.

Yes, the LP's Ledger Rune address holds enough Rune, and all the LPs I tested use this same address.

The keystore address is empty though. Have you tested this scenario?

**Comment by hippocampus-web3** [comment]
This error has been fixed as part of this PR, so it should be possible to close it.

https://github.com/asgardex/asgardex-desktop/pull/732
